Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -697,19 +742,11 @@ namespace OpenInfraPlatform {
 						//	IfcLoop <- IfcEdgeLoop, IfcPolyLoop, IfcVertexLoop
 						const EXPRESSReference<typename IfcEntityTypesT::IfcLoop>& loop = bound->Bound;
 						bool polyOrientation = bound->Orientation;
-						if (!loop) {
-							BLUE_LOG(warning) << "FaceConverter Problem with Face #" << faceID << ": IfcLoop #" << loop->getId() << " no valid loop.";
 
-							if (boundID == 0) 
-							{
-								break;
-							}
-							else 
-							{
-								continue;
-							}
+						if (!loop) {
+							throw oip::InconsistentGeometryException(face, " No valid loop");

```

Comment:
```suggestion
							throw oip::InconsistentModellingException(face, " No valid loop");
```
If the `loop` member variable is not set -> that means that the IFC file was corrupt, not the geometry itself.